docker/gitlab-content-ci/Dockerfile

```diff
@@ -34,4 +34,15 @@ RUN  apt-get update -y \
   && gcloud --version \
   && gsutil --version
 
-
+# Installing goenv
+RUN git clone https://github.com/syndbg/goenv.git ~/.goenv \
+  && echo 'export GOENV_ROOT="$HOME/.goenv"' >> ~/.bashrc \
+  && echo 'export PATH="$GOENV_ROOT/bin:$PATH"' >> ~/.bashrc \
+  && echo 'eval "$(goenv init -)"' >> ~/.bashrc \
+  && echo 'export PATH="$GOROOT/bin:$PATH"' >> ~/.bashrc \
+  && echo 'export PATH="$PATH:$GOPATH/bin"' >> ~/.bashrc \
+  && source ~/.bashrc \
+  && goenv install 1.13.8 \
+  && goenv global 1.13.8 \
+  && source ~/.bashrc \
+  && curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.40.1
```

Comment:
better not to install stuff from the master branch of another repo, if there's a branch corresponding to a version then we should use it.

Reply:
There is none - https://github.com/golangci/golangci-lint/branches/all
This is the recommended installation, see https://golangci-lint.run/usage/install/#other-ci